draft-ietf-rtcweb-jsep.xml

```diff
@@ -2133,10 +2133,17 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           <xref target="I-D.ietf-mmusic-sctp-sdp"></xref>, Section 6.1.
           </t>
 
-          <t>As discussed above, the following attributes are included only
-          if the m= section either has a unique address or is associated
-          with the bundle-tag: "a=ice-ufrag",
-          "a=ice-pwd", "a=fingerprint", "a=dtls-id", and "a=setup".
+          <t>As discussed above, the following attributes of category IDENTICAL
+          or TRANSPORT are included only if the data m= section either has
+          a unique address or is associated with the bundle-tag (e.g., if
+          it is the only m= section):
```

Comment:
I'm not sure this parenthetical is helping, because it seems like if it is really the only m= section then it's not going to be associated with the bundle tag, more likely you just wouldn't bundle